README.md

```diff
@@ -0,0 +1,31 @@
+# Gibo
+[gibo][gibo] (short for .gitignore boilerplates) is a shell script to help you easily
+access .gitignore boilerplates from [Github.com][gitignore].
+
+## Build and install
+Assuming *flatpak* and *flatpak-builder* are installed, follow the next steps:
+
+```bash
+$ git clone https://github.com/flathub/com.github.simonwhitaker.Gibo.git
```

Comment:
Please remove those commands from here. The docs has already this.

Reply:
Not quite sure what you mean, that isn't part of the diff against `new-pr`